src/govuk/components/footer/footer.yaml

```
@@ -7,7 +7,7 @@ params:
   - name: visuallyHiddenTitle
     type: string
     required: false
-    description: Title for a meta item section, which defaults to Support links
+    description: Title for a meta item section, which defaults to support links.
```

'Support links' is the default text used for the visually hidden title, so I think this should be:

```suggestion
    description: Title for a meta item section. Defaults to 'Support links'.
```

---

Ah gotcha. I thought it meant something else! Thanks.